nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,269 @@
+|Jinja Templates|Repository that holds Jinja templates to be used to generate intended configs.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+
+## Repository Details
+
+|Field|Explanation|
+|:--|:--|
+|Name|User friendly name for the repo.|
+|Slug|Computer-friendly name for the repo. Auto-generated based on the `name` provided, but you can change it if you wish.|
+|Remote URL|The URL pointing to the Git repo. Current git url usage is limited to `http` or `https`.|
+|Branch|The branch in the Git repo to use. Defaults to `main`.|
+|Token|The token is a personal access token for the `username` provided.  For more information on generating a personal access token see the corresponding links below. **Token is the only option supported currently.**|
+|Username|The Git username that corresponds with the personal access token above. Note not required for GitHub, but is for GitLab.|
+|Provides|Resource type(s) provided by this Git repo.|
+
+- [GitHub Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
+- [GitLab Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
+- [Bitbucket Personal Access Token](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html)
+
+
+## Using Git Data Sources
+This section will focus on examples and use the `user-guide` branch on the demo-git-datasources repo: `https://github.com/nautobot/demo-git-datasource`.
+
+### Export Templates
+
+[Export Templates](../models/extras/exporttemplate.md) allow a user to export Nautobot objects based on a custom template.  Export templates can change over time depending on the needs of a user.  Allowing export templates to reference a Git repo makes managing templates easier.
+
+A template can be used to put objects into a specific format for ingestion into another system, tool, or report.  It is possible that different templates are needed depending on specific users or teams.  This can lead to sprawl of export templates.  To keep accurate templates synced with Nautobot the Git Data Sources extensibility feature can be used.
+
+#### Step 1
+Navigate to the Data Sources Git integration. **Extensibility -> Git Repositories**.
+
+![](./images/git-data-source_1.png)
+
+Click [+] or [Add]
+
+That loads a default page to add a repository.
+
+![](./images/git-data-source_2.png)
+
+!!! note
+    By default only config contexts, export templates, and jobs are available resource types.  Others may be added when specific plugins are used.
+
+#### Step 2
+Fill out the details for the Git repository. More information on the inputs can be found in the [fields section](#repository-details).
+
+![](./images/git-data-source_3.png)
+
+As soon as you click on **Create**, Nautobot will clone and sync the repository and provide status of the job.
+
+![](./images/git-data-source_4.png)
+![](./images/git-data-source_6.png)
```

Comment:
Looking at the new, narrower versions of these images, we *really* should fix how we render timestamps when showing JobResult logs:

![image](https://user-images.githubusercontent.com/5603551/112547424-1c546780-8d91-11eb-9610-54bb05c62277.png)